src/components/Widgets/ImageOptimize/imageOptimize.js

```diff
@@ -0,0 +1,20 @@
+//optimize images according to view port changes.
+import { breakpoints } from '../../../../config';
+
+class ImageOptimize {
+    constructor(props){
+        this.props = props;
+    }
+
+    optimizeImage(){
+        if(this.props.viewport== "Mobile"){
```

Comment:
Do not leave anything hardcoded. Import everything from libs.

Reply:
Fixed